lib/components/channel_panel.dart

```diff
@@ -254,6 +262,18 @@ class _ChannelPanelWidgetState extends State<ChannelPanelWidget> {
           })
       : Container();
 
+  Widget _buildCommandBar(BuildContext context) {
+    final commandsModel = Provider.of<CommandsModel>(context, listen: false);
```

Comment:
Use Consumers instead of Providers in the `build` functions:

```
Consumer<CommandModel>(builder: (context, commandModel, child) {
   ...
})
```